addons/web/static/src/legacy/scss/bootstrap_overridden.scss

```diff
@@ -168,15 +168,15 @@ $list-group-active-color: $o-list-group-active-color !default;
 $list-group-active-bg: $o-list-group-active-bg !default;
 
 // Forms
-$custom-control-indicator-border-color: $gray-500 !default;
-$custom-control-label-disabled-color: $o-main-color-muted !default;
+$form-check-indicator-border-color: $gray-500 !default;
+$form-check-label-disabled-color: $o-main-color-muted !default;
 
-$custom-control-indicator-checked-color: $white !default;
-$custom-control-indicator-checked-bg: $primary !default;
-$custom-control-indicator-checked-border-color: $primary !default;
+$form-check-indicator-checked-color: $white !default;
+$form-check-indicator-checked-bg: $primary !default;
+$form-check-indicator-checked-border-color: $primary !default;
 
-$custom-control-indicator-disabled-bg: $gray-200 !default;
-$custom-control-indicator-checked-disabled-bg: $gray-200 !default;
+$form-check-indicator-disabled-bg: $gray-200 !default;
+$form-check-indicator-checked-disabled-bg: $gray-200 !default;
```

Comment:
It seems that old variable don't exist anymore in Bootstrap 5.1.
https://getbootstrap.com/docs/5.1/forms/checks-radios/#variables